po/common/uk.po

```diff
@@ -4239,7 +4239,7 @@ msgstr "Замініть наявні значення та фотографії
 
 msgctxt "move_data_and_photos_to_main_language_ignore"
 msgid "Keep existing values and selected photos"
-msgstr "Дотримуйтесь існуючих значень та вибраних фотографій"
```

Comment:
@teolemon This is from the translation widget again, given the number of errors I've seen from this widget, I would be in favor of shutting it down